python/mxnet/libinfo.py

```diff
@@ -76,6 +76,33 @@ def find_lib_path():
         os.environ['PATH'] = os.environ['PATH'] + ';' + os.path.dirname(lib_path[0])
     return lib_path
 
+def find_include_path():
+    """Find MXNet dynamic library files.
+     Returns
+    -------
+    incl_path : string
+        Path to the header files.
+    """
+    incl_from_env = os.environ.get('MXNET_INCLUDE_PATH')
+    if incl_from_env:
+        if os.path.isfile(incl_from_env):
+            if not os.path.isabs(incl_from_env):
+                logging.warning("MXNET_INCLUDE_PATH should be an absolute path, instead of: %s",
+                                incl_from_env)
+            else:
+                if os.name == 'nt':
```

Comment:
May be good to add some comments to explain why we need to update the 'PATH' env variable for Windows.

Reply:
It follows the same usage as in find_lib_path(). I think it's because Windows requires a concatenated path by ;